knowledge_repo/converters/ipynb.py

```diff
@@ -1,11 +1,23 @@
 import os
+import nbconvert
 
 from ..converter import KnowledgePostConverter
 from .._version import __optional_dependencies__
 
 TEMPLATE = '''
 {%- extends 'markdown.tpl' -%}
 
+{%- block data_javascript scoped %}
+{% set div_id = uuid4() %}
+<div id="{{ div_id }}"></div>
+<div class="output_subarea output_javascript {{ extra_class }}">
+<script type="text/javascript">
```

Comment:
What `element` are you referring to?  Do you mean the `extra_class` ? This block of code was taken from the `nbconvert` template for HTML. We are free to change this of course but I felt like we should render it as closely as `nbconvert` does for HTML. It could be that `extra_class` gets pulled from cell metadata (total guess!) before the template is rendered.